README.md

```diff
@@ -82,6 +82,10 @@ Inputs (all of them required):
 - `repository-list` 
```

Comment:
Replace `Inputs (all of them required):` with `Required inputs:` at line 80 above?